.github/renovate.json

```diff
@@ -0,0 +1,21 @@
+{
+	"$schema": "https://docs.renovatebot.com/renovate-schema.json",
+	"packageRules": [
+		{
+			"groupName": "CI and devDependencies",
+			"matchManagers": ["github-actions", "pre-commit"]
+		},
+		{
+			"groupName": "Runtime and test",
+			"matchManagers": ["pep621"]
+		}
+	],
+	"separateMajorMinor": false,
+	"extends": [
+		"config:recommended",
+		":dependencyDashboard",
```

Comment:
This is optional, a similar interface is available at the developer page. https://developer.mend.io/github/LecrisUT/tmt